runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -2382,6 +2396,70 @@ MM_CopyForwardScheme::scanOwnableSynchronizerObjectSlots(MM_EnvironmentVLHGC *en
 	scanMixedObjectSlots(env, reservingContext, objectPtr, reason);
 }
 
+/**
+ *  Iterator the slot reference and parse and pass leaf bit of the reference to copy forward
+ *  to avoid to push leaf object to work stack in case the reference need to be marked instead of copied.
+ */
+MMINLINE bool
+MM_CopyForwardScheme::iteratorAndCopyforwardSlotReference(MM_EnvironmentVLHGC *env, MM_AllocationContextTarok *reservingContext, J9Object *objectPtr) {
```

Comment:
rename it to iterateAndCopyForwardSlots (use verb, rather than noun)

Reply:
done